HomeWork/src/main/java/HomeWork5/HomeWork5.java

```diff
@@ -55,41 +54,47 @@ public static void guessTheWordGame () {
 
     public static String stringReverse (String source) {
         if (source == null) {
-            return "Ошибка!";
+                throw new NullPointerException ("Error: you didn't enter any data." + "\n" +
+                        "null was passed to the argument 'source'.");
         } else {
```

Comment:
You can get rid of the else statement because you have your validation inside the if block.